README.md

```diff
@@ -41,12 +41,13 @@ You also need to add a dependency on the `tf_transformations` in your `package.x
 
 ## Installation
 
-To use this package, you need to manually install the `transforms3d` library.
+This package is available in full as a ROS binary, which you can install via
 
-    sudo pip3 install transforms3d
+    sudo apt install ros-$ROSDISTRO-tf-transformations
+
+In older versions, you would need to manually install the `transforms3d` library via pip.
 
-At the present time, you cannot install `transforms3d` via Ubuntu debian, nor can you force the installation of the
- `tf_transformations` package to also include `transforms3d`.
+    sudo pip3 install transforms3d
```

Comment:
@DLu I had an issue yesterday, and it appears that the library expects explicit version of `transforms3d`, I have ended up having version `0.4.2` on Ubuntu 22.04 (within Docker).

This caused the issue that `tf_transforms` didn't recognize the installation. Then I have installed explicit the version `0.4.1` and everything worked.

```suggestion
    sudo pip3 install transforms3d==0.4.1
```

Reply:
This should not be the case.
Can you provide a minimal dockerfile which has this issue?

I believe the issue might come from the fact that you actually did install it via pip at one point in time.